lms/djangoapps/course_api/blocks/api.py

```diff
@@ -106,7 +107,12 @@ def get_blocks(
 
     # transform
     blocks = course_blocks_api.get_course_blocks(
-        user, usage_key, transformers, allow_start_dates_in_future=allow_start_dates_in_future)
+        user,
+        usage_key,
+        transformers,
+        allow_start_dates_in_future=allow_start_dates_in_future,
+        include_has_scheduled_content=include_has_scheduled_content
```

Comment:
@Dillon-Dumesnil did a similar thing before - is this the right pattern, vs I dunno, adding some new transformer or whatever?

Reply:
Yeah unfortunately. A new transformer doesn't make as much sense because it still has to do with the Start date so it would want to exist in the Start Date Transformer.

Reply:
I thought about adding a new transformer, but I'd have to duplicate or repurpose code from the StartDateTransformer to do so. I'm not sure what the right pattern is, or what the benefit of having a separate transformer might be over including the check for scheduled content in the transformer that filters out scheduled content...